startup/36-pilatus100k.py

```diff
@@ -413,6 +430,12 @@ def describe_collect(self):
         return_dict_trig = self.ext_trigger_device.describe_collect()
         return {**return_dict_pil100k, **return_dict_trig}
 
+    def describe(self):
+        # Add better detector metadata to avoid Tiled server side conversion.
+        res = super().describe()
+        res["pil_100k2"]["dtype_numpy"] = "<i4"
```

Comment:
Marginally better to do 

```suggestion
        res["pil_100k2"].setdefault("dtype_numpy", "<i4")
```

so that if the super class updates this won't start to clobber it.

Reply:
Where is the super class defined?  Just because we _can_ patch it here does not mean we should.  

The dectris detectors are able to output different bit depths so unless we are sure that ISS will never change, we should pull this from the PV (see CHX for an example of the look up logic).

Reply:
Presumably this is what you're talking about:
https://github.com/NSLS-II-CHX/profile_collection/blob/master/startup/20-area-detectors.py#L35-L39
Only for them it's floating point values from dectris PVs... I'll try to make something similar